### ISSUE TITLE
Android do not show tabbaricon in release apk

### ISSUE THREAD

**Issue by jamninetyfive** [okwasniewski/react-native-bottom-tabs#173]
### Before submitting a new issue

- [X] I tested using the latest version of the library, as the bug might be already fixed.
- [X] I tested using a [supported version](https://github.com/reactwg/react-native-releases/blob/main/docs/support.md) of react native.
- [X] I checked for possible duplicate issues, with possible answers.

### Bug summary

Android do not show tabbaricon in release apk. Whether it is svg or png

but it works on debug.

### Library version

0.7.1

### Environment info

```shell
System:
  OS: macOS 15.1.1
  CPU: (10) arm64 Apple M1 Pro
  Memory: 109.27 MB / 16.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 16.20.2
    path: ~/.nvm/versions/node/v16.20.2/bin/node
  Yarn:
    version: 3.6.4
    path: ~/.nvm/versions/node/v16.20.2/bin/yarn
  npm:
    version: 8.19.4
    path: ~/.nvm/versions/node/v16.20.2/bin/npm
  Watchman: Not Found
Managers:
  CocoaPods:
    version: 1.12.1
    path: /Users/yy/.rvm/gems/ruby-2.7.7/bin/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 24.1
      - iOS 18.1
      - macOS 15.1
      - tvOS 18.1
      - visionOS 2.1
      - watchOS 11.1
  Android SDK: Not Found
IDEs:
  Android Studio: 2024.1 AI-241.18034.62.2412.12266719
  Xcode:
    version: 16.1/16B40
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 21.0.1
    path: /usr/bin/javac
  Ruby:
    version: 2.7.7
    path: /Users/yy/.rvm/rubies/ruby-2.7.7/bin/ruby
npmPackages:
  "@react-native-community/cli":
    installed: 15.1.2
    wanted: 15.1.2
  react:
    installed: 18.3.1
    wanted: 18.3.1
  react-native:
    installed: 0.76.3
    wanted: 0.76.3
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: true
iOS:
  hermesEnabled: true
  newArchEnabled: true
```

### Steps to reproduce

android only

### Reproducible sample code

```js
import {createNativeBottomTabNavigator} from '@bottom-tabs/react-navigation';

const HOMEICON = require('../shared/icons/home.png')
const BottomTab = createNativeBottomTabNavigator();

<BottomTab.Navigator initialRouteName="Home" ignoresTopSafeArea={false} hapticFeedbackEnabled barTintColor={'#ffffff'}>
      <BottomTab.Screen
        name="Home"
        component={HomeScreen}
        options={{
          title: 'Home',
          tabBarIcon: () => HOMEICON,
        }}
      />
</BottomTab.Navigator>
```

**Comment by Peek-A-Booo** [comment]
the same problem. I just made a [pr](https://github.com/okwasniewski/react-native-bottom-tabs/pull/172)🤣

**Comment by okwasniewski** [comment]
Fixed in 0.7.2
